openpype/hosts/maya/plugins/publish/validate_rig_contents.py

```diff
@@ -144,7 +150,84 @@ def validate_controls(self, set_members):
         # Validate control types
         invalid = []
         for node in set_members:
-            if cmds.nodeType(node) not in self.accepted_controllers:
+            if cmds.nodeType(node) not in cls.accepted_controllers:
                 invalid.append(node)
 
         return invalid
+
+    @classmethod
+    def get_nodes(cls, instance):
```

Comment:
The return values, what it should return, isn't obvious at first so it might be good to add a decent docstring for these methods.